src/main/io/vtx.c

```diff
@@ -92,7 +92,7 @@ void vtxInit(void)
     }
 }
 
-static vtxSettingsConfig_t vtxGetSettings(void)
+vtxSettingsConfig_t vtxGetSettings(void)
```

Comment:
I assume this is only for unit testing? We have a `STATIC_UNIT_TESTED` macro for that (plus add the declaration to the test itself instead of to the header).

Reply:
Ah nice, changed!